data/units.yaml

```diff
@@ -0,0 +1,6 @@
+gravimetric:
+  label: Gravimetric Units
+  unit: ms^-2
+euros_millions:
+  label: Euros (Millions)
+  unit: €m
```

Comment:
I was hoping to uniform these to the actual value and just use "Euro" (same for all currencies), using millions can be easily be formatted in the UI based on value ranges

Reply:
Fine with me. We'll need to make sure that everyone is aware of this, as I think that most datasets will need some transformation applied